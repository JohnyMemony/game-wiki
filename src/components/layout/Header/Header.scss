@import "src/assets/styles/variables";

.header {
  background-color: $dark;

  &__content {
    display: flex;
    height: 80px;
    align-items: center;
  }

  &__logo {
    width: 40px;
    height: 40px;
  }
}
