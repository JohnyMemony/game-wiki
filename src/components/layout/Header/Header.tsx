import React from 'react';
import { Link } from 'react-router-dom';
import Logo from '../../../assets/images/svg/game-controller.svg';
import { ROUTES } from '../../../constants/routes';

import './Header.scss';

export const Header: React.FC = () => {
  return (
    <header className="header">
      <div className="container">
        <div className="header__content">
          <Link to={ROUTES.HOME}>
            <img className="header__logo" src={Logo} alt="" />
          </Link>
        </div>
      </div>
    </header>
  );
}
