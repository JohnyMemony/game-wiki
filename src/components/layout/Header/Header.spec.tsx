import React from 'react';
import { Header } from './Header';
import { shallow } from 'enzyme';

describe('Header component', () => {
  it('should render Header component', function () {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
