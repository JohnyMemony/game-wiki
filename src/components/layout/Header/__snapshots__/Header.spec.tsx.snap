// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header component should render Header component 1`] = `
<header
  className="header"
>
  <div
    className="container"
  >
    <div
      className="header__content"
    >
      <Link
        to="/"
      >
        <img
          alt=""
          className="header__logo"
          src={
            Object {
              "process": [Function],
            }
          }
        />
      </Link>
    </div>
  </div>
</header>
`;
